### PR TITLE
Unit Tests problem 8

### PR DESCRIPTION
**Pull request by millro04**

### CODE REVIEW

**Comment by bnmnetp** [comment]
OK, now do some more

File: _sources/Recursion/ProgrammingExercises.rst [hunk 1]
```diff
@@ -14,10 +14,45 @@ Programming Exercises
...
+          def testOne(self):
+              self.assertEqual(computeFactorial(0),1,"Your function failed with input 0")
+              self.assertEqual(computeFactorial(1),1,"Your function failed with input 1")
+              self.assertEqual(computeFactorial(2),2,"Your function failed with input 2")
+              self.assertEqual(computeFactorial(3),6,"Your function failed with input 3")
+              self.assertEqual(computeFactorial(4),24,"Your function failed with input 4")
+              self.assertEqual(computeFactorial(8),40320,"Your function failed with input 8")
+
+      myTests().main()
+
```
Comment (bnmnetp) [inline _sources/Recursion/ProgrammingExercises.rst]: How about testing a negative number?

**Comment by bnmnetp** [comment]
Are all of these changes combined into Devin's pull request?  If so we should close this one.

**Comment by millro04** [comment]
No it doesn't look like these have been combined into his new pr yet. I'll send these changes to him so he can add them and then we can close this one.

**Pull request closed without merging**